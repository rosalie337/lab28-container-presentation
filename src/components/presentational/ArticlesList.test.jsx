import React from "react";
import { render, cleanup } from "@testing-library/react";
import ArticleList from "./ArticlesList.jsx";

describe("ArticleList component", () => {
  afterEach(() => cleanup());
  it("renders a list of Articles", () => {
    const { asFragment } = render(<ArticleList articles={[]} />);
    expect(asFragment()).toMatchSnapshot();
  });
});