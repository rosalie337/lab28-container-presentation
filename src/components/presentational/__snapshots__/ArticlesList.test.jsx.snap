// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleList component renders a list of Articles 1`] = `
<DocumentFragment>
  <ul />
</DocumentFragment>
`;
